packages/grafana-data/src/utils/location.ts

```diff
@@ -43,6 +51,16 @@ const stripBaseFromUrl = (urlOrPath: string): string => {
     segmentToStrip = `${window.location.origin}${appSubUrl}`;
   }
 
+  // strip bases of some problematic relative links
+  if (!isAbsoluteUrl){
+    for (const segment in problematicRelativeLinks){
+      if (urlOrPath.startsWith(segment)){
+        segmentToStrip =  segment;
+        break;
+      }
+    }
+  }
```

Comment:
@Letra103622 I don't think this logic could ever be correct, as this just removes the relative segments from a URL, instead of somehow resolving the URL to the correct absolute path.

For example, if the user is currently on `/a/b/c/d/e/f`, and they click on a link to `../g`, we would expect them to do to `/a/b/c/d/e/g`, but this logic would take them to `/g` wouldn't it?

Reply:
Hi @joshhunt, thanks for the input.
It's indeed true that the logic isn't correct. My adjustments were aimed at aligning the application with the current browsing mechanism. For some reason, all links seem to direct to the same page and since I couldn't pinpoint the specific code responsable for the link browsing functionality, I  decided to adapt the application to mirror the existing browsing behavior. I also didn't know if it was in the apps best interest to change something that probably is defined in some inported framework.
Please let me know if you have a sugestion for an alternative course of action that we could take.